LPD-53971

rachaelkoestartyo opened the pull request.

Commit:
LPD-52625 Added Padding for Metadata Handling

Mikellorza committed:
LPD-52010 Add collaborators headless api for object entry folders

Mikellorza committed:
LPD-52010 Implement rest endpoints

Mikellorza committed:
LPD-52010 Add tests

juanjofgliferay committed:
LPD-51035 - Refactor. Get URL after CX is registered

Mikellorza committed:
LPD-52010 Add collaborators headless api for object entries

Mikellorza committed:
LPD-52010 buildRest

Mikellorza committed:
LPD-52010 Implement object entry rest endpoints

Mikellorza committed:
LPD-52010 Delete osgi component annotation because we will register it dynamically for each definition

Mikellorza committed:
LPD-52010 Add tests

Mikellorza committed:
LPD-52010 Add new method to delete a sharing entry

Mikellorza committed:
LPD-52010 buildService

Mikellorza committed:
LPD-52010 Add new method for get sharing entries

Mikellorza committed:
LPD-52010 Add sharing entry api tests

Mikellorza committed:
LPD-52010 Refactor and simplify

daiazius committed:
LPD-44396 Add render method to the Captcha interface

daiazius committed:
LPD-44396 Use new render method in CaptchaTag and delete page.jsp, as we're not using it anymore

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/captcha/Captcha.java

Removed getPage in favor of overriding doEndTag to render the captchas.

## Why

We're now using the new render method in the interface to render captchas, so there's no need to have the getPage method.

----

daiazius committed:
LPD-44396 Implement new method in SimpleCaptchaImpl

daiazius committed:
LPD-44396 Implement new method in FunctionCaptchaImpl to render a captcha Custom Element CX through its external reference code

Mikellorza committed:
LPD-52010 Register collaborator resource dynamically for each object definition

georgel-pop-lr committed:
LPD-52738 Fix name export

georgel-pop-lr committed:
LPD-52738 Add Space List fragment in the site-initializer-cms module

georgel-pop-lr committed:
LPD-52738 Add a playwright test for Space List CMS in fragments.spec.ts

daiazius committed:
LPD-44396 Add test

daiazius committed:
LPD-44396 semver

daiazius committed:
LPD-44396 Fix test

daiazius committed:
LPD-44396 ant update-portal-osgi-configuration-properties

joshuacords committed:
LPD-10653 LPD-44542 search-experiences-service: put subtype filtering inside a FF

https://liferay.atlassian.net/browse/LPD-44542

oliv-yu committed:
LPD-23000 LPD-45838 search-experiences-web: Keep selected subtypes behind LPS-129412

joshuacords committed:
LPD-10653 LPD-45837 search-experiences-web: update copyright

https://liferay.atlassian.net/browse/LPD-45837

ling-alan-huang committed:
LPD-52616 Rework YML checks

ling-alan-huang committed:
LPD-52616 Fix test cases

ling-alan-huang committed:
LPD-52616 Format quotes

ling-alan-huang committed:
LPD-52616 Fix regex for decimal

ling-alan-huang committed:
LPD-52616 Fix broken test cases

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 Fix regex

ling-alan-huang committed:
LPD-52616 Sort mountPaths

ling-alan-huang committed:
LPD-52616 Add test case

ling-alan-huang committed:
LPD-52616 Fix

BarbaraCabrera committed:
LPD-53599 Speed up transition

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 SF

MrJohn1911 committed:
LPD-53463 Persist the custom permissions from the PUT request

MrJohn1911 committed:
LPD-53462 Persist custom permissions from the POST request

georgel-pop-lr committed:
LPD-52738 Import {createRoot} from 'react-dom/client'; for React 18

MrJohn1911 committed:
LPD-53462 Add test for POST method

daiazius committed:
LPD-44369 Remove getTaglibPath as we're not using it anymore

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/captcha/Captcha.java

Removed the getTagLibPath method from the interface.

## Why

To clean up the interface as we're not using it anymore. This method is implemented directly in the Captcha implementations.

----

MrJohn1911 committed:
LPD-53463 Add test for PUT method

joshuacords committed:
LPD-10653 LPD-45837 search-experiences-rest-api: baseline

https://liferay.atlassian.net/browse/LPD-45837

joshuacords committed:
LPD-10653 LPD-45838 portal-language-lang: buildLang

https://liferay.atlassian.net/browse/LPD-45838

joshuacords committed:
LPD-10653 LPD-45837 search-experiences-web: make names consistent

https://liferay.atlassian.net/browse/LPD-45837

joshuacords committed:
LPD-23000 LPD-45838 search-experiences-web: make names consistent

https://liferay.atlassian.net/browse/LPD-45838

ling-alan-huang committed:
LPD-52616 Add more boolean values

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 Standardize on true/false

ling-alan-huang committed:
LPD-52616 SF

BarbaraCabrera committed:
LPD-53599 Fix padding

Mikellorza committed:
LPD-52010 Extract common logic to a CollaboratorUtil and use it from CollaboratorResourceImpl classes

ling-alan-huang committed:
LPD-52616 Add test case

ling-alan-huang committed:
LPD-52616 Remove duplicate tests

ling-alan-huang committed:
LPD-52616 Not needed

BarbaraCabrera committed:
LPD-53599 Remove not needed CSS

BarbaraCabrera committed:
LPD-53599 Remove bottom margin

slnn committed:
LPD-54143 Rename the property with more explicit name

ling-alan-huang committed:
LPD-52616 Not needed

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 Not needed

oliv-yu committed:
LPD-23000 LPD-45838 search-experiences-web: Fix variable in order to work again

oliv-yu committed:
LPD-23000 LPD-45838 search-experiences-web: Rename to 'assetSubtype' for consistency

BryanEngler committed:
LPD-10653 LPD-44542 document-library-service: rename

https://liferay.atlassian.net/browse/LPD-44542

BryanEngler committed:
LPD-10653 LPD-44542 search-experiences-service: This pattern is more common

https://liferay.atlassian.net/browse/LPD-44542

rachaelkoestartyo committed:
LPD-53971 add new analytics cms modules

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

rafaprax committed:
LPD-44396 SF, as used

rafaprax committed:
LPD-44396 Add log

joshuacords committed:
LPD-10653 LPD-44542 journal-service: rename

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 search-experiences-service: rename

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 portal-search-api: add interface

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 portal-search-api: baseline

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 search-experiences-service: implement

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 search-experiences-service: use AssetSubtypeIdentifier in search request

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 document-library-service: use AssetSubtypeIdentifier from request

https://liferay.atlassian.net/browse/LPD-44542

joshuacords committed:
LPD-10653 LPD-44542 journal-service: use AssetSubtypeIdentifier from request

https://liferay.atlassian.net/browse/LPD-44542

slnn committed:
LPD-54143 Update test

Mikellorza committed:
LPD-52010 buildRest

Mikellorza committed:
LPD-52010 baseline

marcosapmf committed:
LPD-54207 add gradle dependency

marcosapmf committed:
LPD-54207 query by only the portletIds that matters

BryanEngler committed:
LPD-10653 LPD-44542 portal-search-api: Add builder interface

https://liferay.atlassian.net/browse/LPD-44542

BryanEngler committed:
LPD-10653 LPD-44542 portal-search: Implement in portal-search

https://liferay.atlassian.net/browse/LPD-44542

BryanEngler committed:
LPD-10653 LPD-44542 search-experiences-service: Use builder

https://liferay.atlassian.net/browse/LPD-44542

igor-franca committed:
LPD-52535 Update the label with the new content in the playwright test

rafaprax committed:
LPD-44396 Avoid NPE in the first page rendering after the startup when the FreeMarkerFragmentEntryValidator#validateFragmentEntryHTML is invoked with a DummyHttpServletResponse, see https://github.com/liferay/liferay-portal/blob/e8df1588ee0d7b280f1f0ec21f2758d86d4cfdea/modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-freemarker/src/main/java/com/liferay/fragment/entry/processor/freemarker/FreeMarkerFragmentEntryValidator.java#L88

It only happens on the first rendering due to the lazy initialization of service tracker on this line here (https://github.com/liferay/liferay-portal/blob/0a5a4359810d7aa150b678c7eadd2d45b7fe7e31/modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/contributor/FragmentCollectionContributorRegistryImpl.java#L148) that will trigger the fragmentEntry validation process here (https://github.com/liferay/liferay-portal/blob/0a5a4359810d7aa150b678c7eadd2d45b7fe7e31/modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/contributor/FragmentCollectionContributorRegistryImpl.java#L465).

BryanEngler committed:
LPD-10653 LPD-44542 search-experiences-service: Update test

https://liferay.atlassian.net/browse/LPD-44542

rafaprax committed:
LPD-44396 Reuse code

ling-alan-huang committed:
LPD-52616 Remove unused methods

ling-alan-huang committed:
LPD-52616 Consolidate logic

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 SF

ling-alan-huang committed:
LPD-52616 Fix initial capacity for StringBundler

juanjofgliferay committed:
LPD-51035 - Refactor. Remove extra test step.

Removed step was opening language selector and selecting an untranslated language, leading to an empty Name value

BarbaraCabrera committed:
LPD-53599 Fix test "Does not show alert when accessing a page with a web content display map"

vamshikrishnamaddi committed:
LPD-52625 Add Test

Mikellorza committed:
LPD-52010 Update open api json tests

ruben-pulido committed:
LPD-49153 Set the thumbnail when adding a display page template via POST

ruben-pulido committed:
LPD-49153 Set the thumbnail when updating a display page template via PUT

ruben-pulido committed:
LPD-49153 Set the thumbnail when updating a display page template via PATCH

ruben-pulido committed:
LPD-49153 Add test to ensure display page template thumbnail can be set via POST

matyaswollner committed:
LPD-44691 Import Account-Associated Permissions

matyaswollner committed:
LPD-44691 Integration test

ruben-pulido committed:
LPD-49153 Add test to ensure display page template thumbnail can be set via PUT.

It should be possible to set thumbnail both when a new display page template is created via PUT and when an existing one is updated via PUT.

ruben-pulido committed:
LPD-49153 Add test to ensure display page template thumbnail can be removed via PUT.

ruben-pulido committed:
LPD-49153 Add test to ensure display page template thumbnail can be updated via PATCH.

daiazius committed:
LPD-44396 Clean unnecessary attributes

daiazius committed:
LPD-44396 SF inline

igor-franca committed:
LPD-52535 Announce richText label and help message via screen reader

thiagobuarqque committed:
LPD-53123 move test to Playwright

oliv-yu committed:
LPD-23000 LPD-45838 portal-web: Update related blueprint paths

oliv-yu committed:
LPD-23000 LPD-45838 search-experiences-web: Clean up appearance when there are no subtypes in modal

brookedalton committed:
LPD-54289 Use new group

ling-alan-huang committed:
LPD-52616 Rename

ling-alan-huang committed:
LPD-52616 Not needed

ling-alan-huang committed:
LPD-52616 SF

movaldivia committed:
LPD-53848 manage global enums

movaldivia committed:
LPD-53848 add example to rest openapi

movaldivia committed:
LPD-53848 update ftl to manage global

movaldivia committed:
LPD-53848 gw buildRest

movaldivia committed:
LPD-53848 sf

slnn committed:
LPD-54143 Add support to allow users to specify the host name prefix for these non admin instances

movaldivia committed:
LPD-53848 add array enum case

movaldivia committed:
LPD-53848 gw buildRest

ccorreagg committed:
LPD-53848 Keep the current pattern

bryceosterhaus committed:
LPD-54241 - Remove input-move-boxes component and use ClayDualListBox

ccorreagg committed:
LPD-53848 Avoid inserting into the allSchemas map, as we were not inserting them, and instead, it seems in the past we used the getGlobalEnumSchemas methods to return the global enum schemas. As it seems we are mixing enum schemas, create a new method to return the enums from the Map. The getGlobalEnumSchemas method will use the OpenAPI to know those are definitely global

ccorreagg committed:
LPD-53848 Use methods according to the required enums

ccorreagg committed:
LPD-53848 Not needed anymore, we are not adding the global enums to the allSchemas map

ccorreagg committed:
LPD-53848 buildREST

ccorreagg committed:
LPD-53848 Not needed

ccorreagg committed:
LPD-53848 Not needed

bryceosterhaus committed:
LPD-51547 - Move ESM util under the esm package

igor-franca committed:
LPD-52455 Update external reference code in ObjectFolderItem data structure

brbalazs committed:
LPD-44671 Added import of role associated permissions

brbalazs committed:
LPD-44671 Integration test

rachaelkoestartyo committed:
LPD-53971 rest-config and rest-openapi

JhossephG committed:
LPD-54032 Add objectDefinitionId column in ObjectEntryVersion

JhossephG committed:
LPD-54032 buildService

JhossephG committed:
LPD-54032 Update method to use new approach

jonathanmccann committed:
LPD-54034 Use 'detach' instead of 'dispose' as that method does not exist for this object and it was previously being detached

jonathanmccann committed:
LPD-54034 Add Playwright test

Commit:
LPD-8036 Update replacements.json

Commit:
LPD-8036 Add new testcase for UpgradeCatchAllCheck

JhossephG committed:
LPD-54032 Add upgrade step

igor-franca committed:
LPD-52455 Create a playwright test to ensure the object definition remains in its folder

smottal committed:
LPD-54323 Rename RowCheckers

MicaelleSilvaa committed:
LPD-27744 Add a new case to replace XMLUtil to SAXReaderUtil in replacements.json

MicaelleSilvaa committed:
LPD-27744 Add a new testcase to XMLUtil case

marcellustavares committed:
LPD-53681 | LPD-54359 Enable Prisma Cloud Scan Step for the Faro Image

ling-alan-huang committed:
LPD-52616 Generated

ling-alan-huang committed:
LPD-52616 Auto SF

bryceosterhaus committed:
LPD-54311 - Fix submitForm logic since we override this in spa-web

brianchandotcom committed:
LPD-23000 Sort

brianchandotcom committed:
LPD-23000 Interface

brianchandotcom committed:
LPD-23000 Wordsmith

brianchandotcom committed:
LPD-23000 Rename

brianchandotcom committed:
LPD-23000 Wordsmith

brianchandotcom committed:
LPD-23000 Regen

brianchandotcom committed:
LPD-23000 Wordsmith to match modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/search/spi/model/query/contributor/DLFileEntryModelPreFilterContributor.java

brianchandotcom committed:
LPD-23000 Rename

brianchandotcom committed:
LPD-23000 Rename

brianchandotcom committed:
LPD-52616 prep next

brianchandotcom committed:
LPD-52616 prep next

brianchandotcom committed:
LPD-52625 SF

brianchandotcom committed:
LPD-52625 Make a method so that all logic is in one place, and we can trace it if we ever need to

brianchandotcom committed:
LPD-52625 Make it obvious what you are testing

brianchandotcom committed:
LPD-52010 Every test method uses a new instance of this class, so it's already cleared

brianchandotcom committed:
LPD-52010 Rename

brianchandotcom committed:
LPD-52010 Does this matter?

brianchandotcom committed:
LPD-52010 SF

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPD-52010 Inline

brianchandotcom committed:
LPD-52010 Sort

brianchandotcom committed:
LPD-52010 Sort

brianchandotcom committed:
LPD-52010 As used

brianchandotcom committed:
LPD-52010 SF

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

brianchandotcom committed:
LPD-54388 Typo, should be plural

brianchandotcom committed:
LPD-54264 Publish to fix error when running gw buildREST in modules:

$ gw buildREST
Configuration on demand is an incubating feature.

FAILURE: Build failed with an exception.

* Where:
Build file '/home/me/dev/projects/liferay-portal/modules/third-party/javax-portlet/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':third-party:javax-portlet'.
   > Could not find com.liferay:com.liferay.portal.tools.jakarta.ee.transformer:1.0.0.
     Searched in the following locations:
       - file:/home/me/dev/projects/liferay-portal/.m2/com/liferay/com.liferay.portal.tools.jakarta.ee.transformer/1.0.0/com.liferay.portal.tools.jakarta.ee.transformer-1.0.0.pom
       - https://repository-cdn.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal.tools.jakarta.ee.transformer/1.0.0/com.liferay.portal.tools.jakarta.ee.transformer-1.0.0.pom
     Required by:
         unspecified:unspecified:unspecified

* Try:

annasuszter committed:
LPD-54367 remove the npm-angular-portlet project template because it is deprecated

brianchandotcom committed:
LPD-54264 Regen

brianchandotcom committed:
LPD-54264 prep next

brianchandotcom committed:
LPD-54264 prep next

brianchandotcom committed:
LPD-53848 Can I sort this?

brianchandotcom committed:
LPD-53848 prep next

brianchandotcom committed:
LPD-53848 prep next

brianchandotcom committed:
LPD-44396 Rename

brianchandotcom committed:
LPD-44396 Rename

brianchandotcom committed:
LPD-44396 Inline

brianchandotcom committed:
LPD-44396 SF

brianchandotcom committed:
LPD-44396 SF

brianchandotcom committed:
LPD-44396 Publish gradle-plugins-workspace

brianchandotcom committed:
LPD-44396 Regen

ccorreagg committed:
LPD-53848 Null safe

brianchandotcom committed:
LPD-53848 prep next

brianchandotcom committed:
LPD-53848 prep next

brianchandotcom committed:
U135

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

Commit:
LCD-46211 Fix broken references in terraform

ling-alan-huang committed:
LPD-52625 Auto SF

BryanEngler committed:
LPD-10653 LPD-44542 search-experiences-web: Forgot to apply builder here

https://liferay.atlassian.net/browse/LPD-44542

ling-alan-huang committed:
LPD-52010 Auto SF

BryanEngler committed:
LPD-10653 LPD-44542 search-experiences-web: Rename

https://liferay.atlassian.net/browse/LPD-44542

daiazius committed:
LPD-44396 Fix inline

nicolas-sss committed:
LPD-27744 Improve UpgradeCatchAll replacement to do replacements by chained calls

Mikellorza committed:
LPD-54425 SF because of LPD-52616 changes

brianchandotcom committed:
LPD-54367 prep next

brianchandotcom committed:
LPD-54359 SF

ling-alan-huang committed:
LPD-43765 Not needed

michaelhashimoto committed:
LRCI-5774 Move JenkinsConsoleTextLoader constructor to be private

ling-alan-huang committed:
LPD-43765 Add dependencies

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 Add dependencies

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

Commit:
LCD-46219 Use a randomly generated bucket name to avoid collisions

ling-alan-huang committed:
LPD-43765 Add dependencies

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 Remove unused dependencies

ling-alan-huang committed:
LPD-43765 Rename

carolmariaabb committed:
LPD-54432 Use setBoolean instead

Same as 4e96be7040e656a74114fe58104d5bfded5e6c5e

shuyangzhou committed:
LPD-54452 Rename existing org-apache-aries-cdi modules, preparing for jakarta patching

shuyangzhou committed:
LPD-54452 Jakarta patching aries cdi modules.

daiazius committed:
LPD-44396 Refactor BaseCaptchaTestCase into a util

brianchandotcom committed:
LRCI-5774 prep next

brianchandotcom committed:
LPD-54452 Bump bnd.bnd because those were already published

brianchandotcom committed:
LPD-54452 prep next

brianchandotcom committed:
LPD-54452 auto apply, but I will revert just to show

brianchandotcom committed:
Revert "LPD-54452 auto apply, but I will revert just to show"

This reverts commit 1b917f807c78cc496d04f45ebd65510d3573c00c.

brianchandotcom committed:
LPD-10653 LPD-44542 Sort

brianchandotcom committed:
LPD-52455 Sort

brianchandotcom committed:
LPD-54143 Rename

brianchandotcom committed:
LPD-54143 Rename

brianchandotcom committed:
LPD-54143 auto SF

liferay-translation committed:
LPS-77699 Update Translations

rachaelkoestartyo committed:
LPD-53971 buildREST
